cmake/modules/BuildRocksDB.cmake

```diff
@@ -60,15 +61,23 @@ function(build_rocksdb)
   set(rocksdb_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/rocksdb")
   set(rocksdb_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/rocksdb")
   set(rocksdb_LIBRARY "${rocksdb_BINARY_DIR}/librocksdb.a")
+  if(CMAKE_MAKE_PROGRAM MATCHES "make")
+    # try to inherit command line arguments passed by parent "make" job
+    set(make_cmd "$(MAKE)")
```

Comment:
i'd sugget put
```cmake
if(CMAKE_MAKE_PROGRAM MATCHES "make")
  # try to inherit command line arguments passed by parent "make" job
  set(make_cmd "$(MAKE) rocksdb")
else()
  ${CMAKE_COMMAND} --build <BINARY_DIR> --target rocksdb
endif()
```
as we don't need to depend on gmake or make unless necessary. which is true when it comes to SPDK/DPDK.